govc/USAGE.md

```diff
@@ -4514,6 +4515,190 @@ Options:
   -s=                    Raw String to Send
   -vm=                   Virtual machine [GOVC_VM]
 ```
+<details><summary>List of aliases for USB HID codes</summary>
```

Comment:
USAGE.md is generated (via `make doc`), you just need to put this list with `keystrokes.Description()`, otherwise the next `make doc` update will overwrite it.

Reply:
that would make the output of govc vm.keystrokes -h
very verbose though.